reference/paths/apple_pay_certificates__add-apple-pay-certificate.yaml

```
@@ -0,0 +1,41 @@
+---
+operationId: add-apple-pay-certificate
+summary: New Apple Pay certificate registration
```

```suggestion
summary: Start registering an Apple Pay certificate
```

All endpoints start with a verb.